addons/event_sale/models/event.py

```diff
@@ -60,6 +73,28 @@ def _is_event_registrable(self):
         else:
             return False
 
+    @api.depends('event_ticket_ids.beginning')
+    def _compute_start_sale_date(self):
+        for event in self:
+            event.start_sale_date = min(ticket.beginning for ticket in event.event_ticket_ids)
```

Comment:
Just for coherency, I would name it the same as the ticket field (see comment on that field, sale_date_start instead of beginning is proposed).

Reply:
@tde-banana-odoo I call it `beginning`, because there was already a field named `deadline` on the event tickets